lib/brod_client.ex

```diff
@@ -238,7 +238,7 @@ defmodule BroadwayKafka.BrodClient do
   defp validate_option(:ssl, value) when is_boolean(value), do: {:ok, value}
 
   defp validate_option(:ssl, value) do
-    if Keyword.keyword?(value) do
+    if supported_keyword?(value) do
```

Comment:
I think in this case it should remain as Keyword. IIRC SSL config expects atom keys.